src/main/kotlin/Day11.kt

```diff
@@ -12,7 +12,7 @@ class Day11 : Data<String?> {
 
     // checking that string shouldn’t be containing the letters i, o, l.
      fun mistakenLetters(word: String?): Boolean {
-        return !Pattern.matches(".*[iol]",word)
+        return !Pattern.matches("[iol]",word)
     }

```

Comment:
The `.*` isn't necessary here since it doesn't matter _where_ any of `i`, `o`, or `l` appear - as long as they're somewhere in the string.